executor_manager/executors/docker/executor.py

```
@@ -609,6 +609,18 @@ def _prepare_docker_command(
         # Convert task to JSON string
         task_str = json.dumps(task)
 
+        # DEBUG: Save original task info to file for comparison
+        debug_file = f"/tmp/task_info_em_{task_id}_{subtask_id}.json"
```

Ruff (0.15.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/task_info_em_"

(S108)